docs/revanced-reddit/wiki/help/troubleshooting.md

```diff
@@ -19,11 +19,11 @@ Whitelist `s.youtube.com` in your ad-blocker.
 
 ## The player UI on YouTube doesn't go away
 
-This issue occurs randomly. Currently, the only fix is to restart the app.
+You may have it set to permanently show in **Settings > Accessibility**. Otherwise, this issue occurs randomly and the only fix is to restart the app.
```

Comment:
```suggestion
You may set it to show in **Settings > Accessibility**permanently. Otherwise, this issue occurs randomly, and currently the only fix is to restart the app.
```